applications/GeoMechanicsApplication/tests/test_sellmeijers_rule_validation.py

```diff
@@ -144,11 +144,57 @@ def critical_head_loop(self, file_path, kappa, d70, Hc, search_type='linear'):
+         ('7.11', 3.00E-04, 1.157E-12, 19.1, 21.9, 10.5, 'test_compare_sellmeijer/HeightAquiferD10L60line'),
+         ('7.12', 3.00E-04, 1.157E-12, 27.54, 32.8, 15, 'test_compare_sellmeijer/HeightAquiferD10L90line'),
+         ('7.13', 3.00E-04, 1.157E-12, 9.01, 9.9, 10.5, 'test_compare_sellmeijer/HeightAquiferD20L30line'),
+         ('7.14', 3.00E-04, 1.157E-12, 16.33, 18.2, 16.5, 'test_compare_sellmeijer/HeightAquiferD20L60line'),
+         ('7.15', 3.00E-04, 1.157E-12, 23.42, 27, 21, 'test_compare_sellmeijer/HeightAquiferD20L90line'),
+         ('7.16', 3.00E-04, 1.157E-12, 7.8, 9.4, 12, 'test_compare_sellmeijer/HeightAquiferD30L30line'),
+         ('7.17', 3.00E-04, 1.157E-12, 15.05, 16.8, 22.5, 'test_compare_sellmeijer/HeightAquiferD30L60line'),
+         ('7.18', 3.00E-04, 1.157E-12, 21.4, 24.1, 28.5, 'test_compare_sellmeijer/HeightAquiferD30L90line'),
+         ('7.19', 1.00E-04, 1.157E-10, 0.74, 0.8, 6, 'test_compare_sellmeijer/HeightAquiferD10L30line'),
+         ('7.20', 1.00E-04, 1.157E-10, 1.37, 1.6, 10.5, 'test_compare_sellmeijer/HeightAquiferD10L60line'),
+         ('7.21', 1.00E-04, 1.157E-10, 1.98, 2.4, 12, 'test_compare_sellmeijer/HeightAquiferD10L90line'),
+         ('7.22', 1.00E-04, 1.157E-10, 0.65, 0.7, 9, 'test_compare_sellmeijer/HeightAquiferD20L30line'),
+         ('7.23', 1.00E-04, 1.157E-10, 1.17, 1.3, 13.5, 'test_compare_sellmeijer/HeightAquiferD20L60line'),
+         ('7.24', 1.00E-04, 1.157E-10, 1.68, 1.9, 15, 'test_compare_sellmeijer/HeightAquiferD20L90line'),
+         ('7.25', 1.00E-04, 1.157E-10, 0.56, 0.6, 6, 'test_compare_sellmeijer/HeightAquiferD30L30line'),
+         ('7.26', 1.00E-04, 1.157E-10, 1.08, 1.2, 16.5, 'test_compare_sellmeijer/HeightAquiferD30L60line'),
+         ('7.27', 1.00E-04, 1.157E-10, 1.54, 1.7, 18, 'test_compare_sellmeijer/HeightAquiferD30L90line'),
+         ('7.28', 3.00E-04, 1.157E-10, 2.22, 2.4, 6, 'test_compare_sellmeijer/HeightAquiferD10L30line'),
+         ('7.29', 3.00E-04, 1.157E-10, 4.12, 4.8, 10.5, 'test_compare_sellmeijer/HeightAquiferD10L60line'),
+         ('7.30', 3.00E-04, 1.157E-10, 5.93, 7.2, 12, 'test_compare_sellmeijer/HeightAquiferD10L90line'),
+         ('7.31', 3.00E-04, 1.157E-10, 1.94, 2.1, 9, 'test_compare_sellmeijer/HeightAquiferD20L30line'),
+         ('7.32', 3.00E-04, 1.157E-10, 3.52, 4, 13.5, 'test_compare_sellmeijer/HeightAquiferD20L60line'),
+         ('7.33', 3.00E-04, 1.157E-10, 5.05, 5.9, 21, 'test_compare_sellmeijer/HeightAquiferD20L90line'),
+         ('7.34', 3.00E-04, 1.157E-10, 1.68, 2, 9, 'test_compare_sellmeijer/HeightAquiferD30L30line'),
+         ('7.35', 3.00E-04, 1.157E-10, 3.24, 3.7, 22.5, 'test_compare_sellmeijer/HeightAquiferD30L60line'),
+         ('7.36', 3.00E-04, 1.157E-10, 4.61, 5.3, 27, 'test_compare_sellmeijer/HeightAquiferD30L90line')])
+    def test_sellmeijers_rule_height_line(self, name, d70, kappa, h_c, h_n, length_n, test_name_line):
```

Comment:
Looking at the new tests, I was thinking about the few DGeoFlow C++ integration tests we have, we might want to add something there as well, but it could be that it's not necessary (at least not in this PR)

Reply:
Created an issue on the backlog to not forget this, but resolving this conversation